pkgs/servers/web-apps/snipe-it/02-display-framework-errors.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php b/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php
+index 04f0e4a..2a99071 100644
+--- a/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php
++++ b/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php
+@@ -115,10 +115,14 @@ class Kernel implements KernelContract
+
+             $response = $this->sendRequestThroughRouter($request);
+         } catch (Exception $e) {
++            die("{$e->getMessage()} {$e->getTraceAsString()}");
```

Comment:
Why is this an improvement over the framework's exception handler? And I'm not sure if we want to leak call traces and stuff like this in a production environment.

Reply:
I disable this patch by default because I agree, we would not want to leak call traces like this. 

However, if Laravel fails to bootstrap itself (due to a syntax error in the configuration file, or missing PHP extension, or incorrect database credentials) then that manifests itself as an exception that only says `class "Log" not found`, which is really painful to debug.

This patch implements the solution [from this blog post](https://sebastiandedeyne.com/debugging-the-dreaded-class-log-does-not-exist-error-in-laravel/). 

There are lots of bug reports like laravel/framework#11724 where users are struggling to work out why their application is failing - this is more problematic when writing a Nix package for a Laravel app because we're necessarily patching and doing strange things that will cause these sorts of failures until the package is correct.